doc/releases/release-notes-3.6.rst

```diff
@@ -302,6 +302,15 @@ API
 Bindings
 ========
 
+* NXP Semiconductors (nxp):
+
+  * Modified bindings:
+
+    * :dtcompatible:`nxp,pcf857x`:
+
+      KConfig option ``CONFIG_GPIO_PCF8574`` is now deprecated.
+      Use ``CONFIG_GPIO_PCF857X`` instead.
```

Comment:
it's not deprecated but rather removed entirely, and so is the dtcompatible. This needs guidance from others as to how to best handle. BTW I think the "Bindings" section in the release notes is auto-populated anyway, so it's rather an entry in the "Migration guide". that would be needed, to explain users what they're expected to do.

Reply:
@kartben The migration guide is added. please have a look and give some suggestions